development/lib/run-command.js

```diff
@@ -15,15 +15,18 @@ const spawn = require('cross-spawn');
  * @param {Array<string>} [args] - The arguments to pass to the command
  * @returns {Array<string>} Lines of output received via STDOUT
  */
-async function runCommand(command, args) {
+async function runCommand(command, args, options = {}) {
   const output = [];
   let mostRecentError;
   let errorSignal;
   let errorCode;
   const internalError = new Error('Internal');
   try {
     await new Promise((resolve, reject) => {
-      const childProcess = spawn(command, args, { encoding: 'utf8' });
+      const childProcess = spawn(command, args, {
```

Comment:
What were these changes used for? :thinking:

Reply:
`runCommand` and `runInShell` now take an options object which allows you to configure `spawn`. In this case these options are specified in `ensureXServerIsRunning` to prevent `xset q` from outputting anything to stdout, which was happening here https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/12187/workflows/777baae3-4b51-42c8-8ff8-c4c7ede52861/jobs/329760 and due to this commit https://github.com/MetaMask/metamask-extension/commit/ab16ddf957433ac049d8dd2a3c93adaee9d8151d. I guess changing `runInShell` isn't technically needed here but I updated both for consistency.

Reply:
Ah, I see. I originally used `runInShell` then changed it to `runCommand`. By itself that might prevent output to stdout. Let me try removing these changes and seeing if that still works.

Reply:
`runCommand` does seem to work the way I want it — I'm not getting seeing any output from `xset` in the CircleCI logs. :)

Reply:
Nice!